pygaggle/rerank/transformer.py

```diff
@@ -32,17 +32,23 @@
            'SentenceTransformersReranker']
 
 prediction_tokens = {
-        'castorini/monot5-base-msmarco':          ['▁false', '▁true'],
-        'castorini/monot5-base-msmarco-10k':      ['▁false', '▁true'],
-        'castorini/monot5-large-msmarco':         ['▁false', '▁true'],
-        'castorini/monot5-large-msmarco-10k':     ['▁false', '▁true'],
-        'castorini/monot5-base-med-msmarco':      ['▁false', '▁true'],
-        'castorini/monot5-3b-med-msmarco':        ['▁false', '▁true'],
-        'unicamp-dl/ptt5-base-msmarco-pt-10k':     ['▁não'  , '▁sim'],
-        'unicamp-dl/ptt5-base-msmarco-pt-100k':    ['▁não'  , '▁sim'],
-        'unicamp-dl/ptt5-base-msmarco-en-pt-10k':  ['▁não'  , '▁sim'],
-        'unicamp-dl/mt5-base-multi-msmarco':       ['▁no'   , '▁yes'],
-        'unicamp-dl/mt5-base-en-pt-msmarco':       ['▁no'   , '▁yes']
+        'castorini/monot5-base-msmarco':             ['▁false', '▁true'],
+        'castorini/monot5-base-msmarco-10k':         ['▁false', '▁true'],
+        'castorini/monot5-large-msmarco':            ['▁false', '▁true'],
+        'castorini/monot5-large-msmarco-10k':        ['▁false', '▁true'],
+        'castorini/monot5-base-med-msmarco':         ['▁false', '▁true'],
+        'castorini/monot5-3b-med-msmarco':           ['▁false', '▁true'],
+        'unicamp-dl/mt5-base-en-msmarco',            ['▁no'   , '▁yes'],
```

Comment:
@lhbonifacio , could you please fix this? It should be `:` instead of `,` after the model name